### DISCUSSION TITLE
Picamera2 example, is this useful for other users?

### DISCUSSION THREAD

**Discussion by mastrolinux** [raspberrypi/picamera2#70] [Show and tell]
I just created the following working example:

https://github.com/mastrolinux/raspberry-pi-security-camera-client

It is a condensed guide and simple way to test the new picamera2 client and uploading the image to a server I implemented here https://github.com/mastrolinux/raspberry-pi-security-camera-server

They still need more refinement but please let me know if that is useful to show how others could use picamera2 with a fully working configuration in a real use-case scenario.

**Comment by davidplowman** [comment]
Thanks for this example. I wonder if, moving forward, we might keep a list of 3rd party examples too, besides the very simple ones of our own that are mostly provided as tutorials. We're going to be starting work on proper documentation fairly soon, so that might be a thing to consider.

**Reply by mastrolinux** [reply]
Cool, if you need help with creating such a list, I can contribute.

**Reply by supertobi** [reply]
I think a good start would be if  you could just post your list here.

**Comment by supertobi** [comment]
As the RaspberryPi UI is written gtk it would be nice to have an gtk example too. But that is nothing, that should stop the release.

**Reply by mastrolinux** [reply]
Makes sense, my examples are intended to run even without UI with runlevel 3. But yes, adding a GTK example is a good idea.
